packages/tools/devtools/devtools-view/src/test/jest/VisualTree.test.tsx

```diff
@@ -118,15 +91,9 @@ describe("VisualTreeView component tests", () => {
 			</MessageRelayContext.Provider>,
 		);
 
-		// TODO: Loop the expand button for n-amount of times.
-		const expandButton = await screen.findByTestId("tree-button");
-		await userEvent.click(expandButton);
-
-		await screen.findByText(/test-node-key/);
```

Comment:
It looks like this test was already not working. This text appears regardless of the click event above. I've confirmed that the way the tree component manages expanded/collapsed content doesn't align with the testing pattern here. I've left a TODO to add interactive tests for this component. For now, I've simplified the case to simply verify that the label we're looking for exists.